spec/models/manageiq/providers/openstack/cloud_manager/refresh_spec_common.rb

```diff
@@ -296,9 +309,17 @@ def assert_ems
       expect(@ems.key_pairs.size).to          eq compute_data.key_pairs.count
       security_groups_count = @ems.security_groups.count { |x| x.name != 'default' }
       expect(security_groups_count).to        eq security_groups_count
-      expect(@ems.vms_and_templates.size).to  eq vms_count + images_count
+      if ::Settings.ems.ems_openstack.try(:refresh).try(:inventory_object_refresh) || ::Settings.ems.ems_refresh.try(:openstack).try(:inventory_object_refresh)
```

Comment:
Alternately, fetch_path is usable on Settings as well, so this could be

```ruby
Settings.ems.fetch_path(:ems_openstack, :refresh, : inventory_object_refresh) || Settings.ems.fetch_path(:ems_refresh, :openstack, : inventory_object_refresh)
```

On that note, why are there even 2 ways to configure things?  Pick whichever one you want to be the "official" one (I suggest the former, so you can own the setting in your repo), and create a data migration to convert the other one.

Reply:
Good to know fetch_path is available on Settings. There are two ways to configure things because the latter (incorrect) option was put in first and merged into a release. I put together a PR to support the correct way and the old way to avoid breaking anyone's deployment since I wasn't aware I could use a migration to change things over. Since the PR that implements the new openstack-owned approach wasn't merged yet, I should be able to implement a migration and get this all cleaned up. :)